pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/apps.kruise.io/v1alpha1/DaemonSet/testdata/observed-daemonset-nginx.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: apps.kruise.io/v1alpha1
 kind: DaemonSet
 metadata:
+  annotations:
+    resourcetemplate.karmada.io/generation: "1"
```

Comment:
```suggestion
metadata:
  annotations:
    resourcetemplate.karmada.io/generation: "1"
  generation: 1
```

Reply:
Sorry, I overlooked something, there should be an additional field here, otherwise the interpretStatus operation will not get status.generation.

Reply:
hello @yike, this field `generation` already exists. :joy_cat:

Reply:
> hello @Yike, this field `generation` already exists. 😹

Haha, I think I may have been a little dizzy at the time.
Thanks for your work!

Reply:
@yike21 you're the thirdparty's owner, you can apporve it if you have no other comments.